src/main/scala/com/gu/fastly/Lambda.scala

```diff
@@ -57,10 +58,10 @@ class Lambda {
    *
    * @return whether a piece of content was purged or not
    */
-  private def sendFastlyPurgeRequest(contentId: String, purgeType: PurgeType): Boolean = {
+  private def sendFastlyPurgeRequest(contentId: String, purgeType: PurgeType, serviceId: String = config.fastlyDotcomServiceId): Boolean = {
```

Comment:
I think it is confusing to have a default `serviceId` in this function call - better to make it explicit everywhere.

Reply:
the endpoint which we're purging here only returns live blog blocks. I suspect (but am not certain) that live blog block delete events are treated as updates by capi as it's just an update to the document of the content. However, currently composer doesn't let you delete a block [without first taking it down](https://github.com/guardian/flexible-content/blob/master/flexible-content-apiv2/src/main/scala/com/gu/flexiblecontent/apiv2/dispatcher/ApiDispatcher.scala#L1558) so our logic here is probably reasonable?